chaco/tools/simple_zoom.py

```diff
@@ -70,25 +70,25 @@ class SimpleZoom(AbstractOverlay, ToolHistoryMixin, BaseZoomTool):
 
     # The key press to enter zoom mode, if **always_on** is False.  Has no effect
     # if **always_on** is True.
-    enter_zoom_key = Instance(KeySpec, args=("z",))
+    enter_zoom_key = Instance(KeySpec, args=("z",), transient=True)
 
     # The key press to leave zoom mode, if **always_on** is False.  Has no effect
     # if **always_on** is True.
-    exit_zoom_key = Instance(KeySpec, args=("z",))
+    exit_zoom_key = Instance(KeySpec, args=("z",), transient=True)
 
     # Disable the tool after the zoom is completed?
     disable_on_complete = Bool(True)
 
     # The minimum amount of screen space the user must select in order for
     # the tool to actually take effect.
-    minimum_screen_delta = Int(10)
+    minimum_screen_delta = Int(10, transient=True)
 
     # -------------------------------------------------------------------------
     # Appearance properties (for Box mode)
     # -------------------------------------------------------------------------
 
     # The pointer to use when drawing a zoom box.
-    pointer = "magnifier"
+    pointer = Str("magnifier", transient=True)
```

Comment:
This may be bad if `pointer` is meant to be a class attribute, similar to `stack_index` in `plot_container.py`.

I need to climb up the class hierarchy to see if it was previously defined as a trait.  I remember Kit mentioning this being a point of confusion.  I am not sure what the ideal way to avoid this confusion would be (aside from maybe a comment above saying if this is a class attribute or just setting a trait to a value.  I guess if setting a trait to some value, you could do as I have done here? and explicitly do something like `Str(___)`

Reply:
As a quick sanity check, on master I defined a class attribute `thing = "A"` on `SimpleZoom`.  I then ran:
```
from chaco.tools.simple_zoom import SimpleZoom
SimpleZoom.thing  # I see 'A' printed
SimpleZoom.pointer  # I get an error "type object 'SimpleZoom' has no attribute 'pointer'"
```

So, I believe the change made here is the right one